### DISCUSSION TITLE
Github does not show any of my commits made to my main branches

### DISCUSSION THREAD

**Discussion by jnjoroge13** [community/community#27531] [Projects and Issues] [answered]
Fixed it thanks

**Comment by sawyerbivens** [comment, marked as answer]
have you properly set up your email and name config in your terminal with `git config --user.name "name"` and `git config --user.email "email@gmail.com"`?

**Comment by jfullstackdev** [comment]
why do you say `I commit to my main branch` ? 

it's owned by another user, unless it is your another account

![image](https://user-images.githubusercontent.com/47092464/181674876-27475a8f-e3de-4cce-b41d-7145eda0117b.png)
